.github/workflows/build.yml

```diff
@@ -0,0 +1,17 @@
+name: iOS Build
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+jobs:
+  build:
+    name: Build eXo iOS Project
+    runs-on: macos-latest
```

Comment:
We may need to put the exact traget version as ios 12

Reply:
OK I use Catalina. Please check the compatibility matrix https://github.com/actions/virtual-environments/blob/main/images/macos/macos-10.15-Readme.md#installed-sdks

Reply:
You can set as minimum macOS 10.15 Catalina

Reply:
Yes it is done

Reply:
@hbenali thanks